.github/helper/install.sh

```diff
@@ -46,18 +54,20 @@ cd ./frappe-bench || exit
 sed -i 's/^watch:/# watch:/g' Procfile
 sed -i 's/^schedule:/# schedule:/g' Procfile
 
-if [ "$TYPE" == "server" ]; then sed -i 's/^socketio:/# socketio:/g' Procfile; fi
-if [ "$TYPE" == "server" ]; then sed -i 's/^redis_socketio:/# redis_socketio:/g' Procfile; fi
-
-if [ "$TYPE" == "ui" ]; then bench setup requirements --node; fi
-if [ "$TYPE" == "server" ]; then bench setup requirements --dev; fi
+if [ "$TYPE" == "server" ]; then
+    sed -i 's/^socketio:/# socketio:/g' Procfile;
+    sed -i 's/^redis_socketio:/# redis_socketio:/g' Procfile;
+    bench setup requirements --dev;
+    # install node-sass which is required for website theme test
+    cd ./apps/frappe || exit
+    yarn add node-sass@^7.0.0
+    cd ../..
+fi
 
-# install node-sass which is required for website theme test
-cd ./apps/frappe || exit
-yarn add node-sass@4.13.1
-cd ../..
+if [ "$TYPE" == "ui" ]; then
+    bench setup requirements --node;
+fi
 
+CI=Yes bench build --app frappe &
```

Comment:
Might need to check exit status of this. 😅 

(issues like these are frequent: https://github.com/frappe/frappe/pull/15982 )

Reply:
can use `wait` towards the end of the script to ensure all background processes were successful.

https://tiswww.case.edu/php/chet/bash/bashref.html#index-wait